src/video-grid/VideoTile.module.css

```diff
@@ -60,16 +59,24 @@
   z-index: 1;
 }
 
-.optionsButton {
-  right: 16px;
-  top: 16px;
+.infoBubbleIcon {
+  background-color: rgba(23, 25, 28, 0.85);
```

Comment:
This could stand to be a theme variable as well - it looks to be a non-standard one borrowed from Element's light mode, so this would be a good opportunity to fix that by replacing it with a similar one from dark mode, making it `--background-85` or something